css-view-transitions-1/Overview.bs

```diff
@@ -61,82 +86,346 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
-	This is accomplished by leveraging user-agents ability to persist visual representations of state (i.e. snapshots)
-	and blend them with current DOM state's visual output.
-	The API also allows the animations to be customized via standard CSS animation properties.
+	View Transitions is a feature that allows DOM changes to smoothly animate between states.
 
-	This spec describes the CSS and JS mechanics of the single-page transition API.
+## Separating transitions from DOM updates ## {#separating-transitions}
 
-# Transitions as an enhancement # {#transitions-as-enhancements}
+	Traditionally, creating a transition between two states involves a period where both states are present in the DOM at the same time.
+	In fact, it usually involves creating a specific DOM structure that can contain both states.
+	If one element is "moving" between containers,
+	that element often needs to exist outside of either container for the period of the transition,
+	to avoid clipping from either container or ancestor elements.
 
-	*This section is non-normative.*
+	This extra in-between state often results in UX and accessibility issues,
+	as the structure of the DOM is compromised for a purely-visual effect.
+
+	View transitions avoid this troublesome in-between state by allowing the DOM to switch between states synchronously,
+	then performing a customizable visual transition between the two states in another layer,
+	using a static capture of the old state, and a live capture of the new state.
+
+	These captures are used in a tree of pseudo-elements
+	(detailed in [[#view-transition-pseudos]]),
+	where the old state cross-fades with the new state,
+	while animating from the old to new size and position.
+
+	The developer chooses which elements are captured independently,
+	meaning they can be animated independently,
+	using the 'view-transition-name' CSS property.
+
+## Lifecycle ## {#lifecycle}
+
+	A successful view transition goes through the following phases:
+
+	1. Developer calls <code>document.{{Document/startViewTransition}}({{UpdateCallback|updateCallback}})</code>,
+		which returns <code>viewTransition</code>, a {{ViewTransition}}.
+
+	1. Current state captured as the "old" state.
+
+	1. Rendering paused.
+
+	1. Developer's {{UpdateCallback|updateCallback}} is called,
+		where they update document state.
+
+	1. <code>viewTransition.{{ViewTransition/updateCallbackDone}}</code> fulfills.
+
+	1. Current state captured as the "new" state.
+
+	1. Transition pseudo-elements created.
+		See [[#view-transition-pseudos]] for an overview of this structure.
+
+	1. Rendering unpaused, revealing the transition pseudo-elements.
+
+	1. <code>viewTransition.{{ViewTransition/ready}}</code> fulfills.
+
+	1. Pseudo-elements animate until finished.
+
+	1. Transition pseudo-elements removed.
+
+	1. <code>viewTransition.{{ViewTransition/finished}}</code> fulfills.
+
+	<div id="phases-diagram">
```

Comment:
A step by step diagram! I was originally going to make this a video, but being able to step back & forward seems useful. It also means we can change it without having to create a whole new video.